src/python_testing/TC_OpstateCommon.py

```diff
@@ -1116,7 +1120,14 @@ async def TEST_TC_OPSTATE_BASE_2_5(self, endpoint=1):
             # reboot outside CI environment.
             if not self.is_ci:
                 self.wait_for_user_input(prompt_msg="Restart DUT. Press Enter when ready.\n")
+                # Expire the session and re-establish the subscription
                 self.default_controller.ExpireSessions(self.dut_node_id)
+                if self.check_pics(f"{self.test_info.pics_code}.S.E01"):
+                    # Subscribe to Events and when they are sent push them to a queue for checking later
```

Comment:
```suggestion
                    # Subscribe to Events and when they are received sent push them to a queue for checking later
```

Is this what you meant to type?